html-block-and-inline/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+
+    <h2>by Snuggie Store</h2>
+
+    <p>Price: <s>$19.99</s> $14.99</p>
+
+    <p>
+      Part of our new Americana Collection, this
+      <em>beautiful</em> blue is already a customer favorite!
+    </p>
+
+    <p>Enjoy our complete <b>Freedom of Movement!</b></p>
```

Comment:
You'll want to use `strong` over `b`. `b` only applies bold styling but `strong` is a semantic element, so it will apply that styling & will also change the way the screenreaders voice reads out the text